Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:9
+# Docker build arguments
+ARG SOURCE_DIR=/repo
+ARG ARTIFACT_DIR=/dist
+ARG ANDROID_DIR=/usr/lib/android-sdk
+# Docker run environment
+ENV SOURCE_DIR=/repo
+ENV ARTIFACT_DIR=/dist
+ENV ANDROID_DIR=/usr/lib/android-sdk
+
+# Prepare Debian build environment
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list \
+ && apt-get update \
+ && apt-get install -y mmv wget unzip git \
+ && apt-get install -t stretch-backports -y android-sdk openjdk-8-jdk
+
+# Install Android repository tools
+RUN rm -rf ${ANDROID_DIR}/tools \
+ && wget http://dl-ssl.google.com/android/repository/tools_r25.2.3-linux.zip -O tools.zip \
+ && unzip tools.zip -d ${ANDROID_DIR}/ \
+ && rm -f tools.zip
```

Comment:
I still think we should be able to remove this somehow but that can be for another day.

Reply:
As long as we need to use sdkmanager, we need it :(